vm/src/vm.rs

```diff
@@ -490,6 +492,20 @@ impl VirtualMachine {
         self.new_exception_msg(import_error, msg)
     }
 
+    // TODO: #[track_caller] when stabilized
+    pub fn unwrap_pyresult<T>(&self, result: PyResult<T>) -> T {
```

Comment:
Where is this used?

Reply:
I figured it would be good as a public API, since `unwrap()`ing a `PyResult` doesn't actually show any information for the `Err`.

Reply:
How about adding `#[cfg(build = "debug")]` and doing regular unwrap on `release`?